src/components/loginButtons/GoogleLogin.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+
+import googleSDK from 'services/googleSDK';
+
+import { Box } from 'uikit/Core';
+import DisabledGoogleLogin from './DisabledGoogleLogin';
+
+const COOKIES_NOT_ENABLED = 'Cookies are not enabled in current environment.';
+
+class GoogleButton extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      disabled: false,
+    };
+  }
+
+  async componentDidMount() {
+    try {
+      await googleSDK();
+      global.gapi.signin2.render('googleSignin', {
+        scope: 'profile email',
+        width: 240,
+        height: 40,
+        longtitle: true,
+        theme: 'light',
+        onsuccess: googleUser => {
+          const { id_token } = googleUser.getAuthResponse();
+          this.props.login(id_token);
```

Comment:
@ciaranschutte I have a feeling the ambiguous reference to `this` here is causing the issue